Makefile

```diff
@@ -10,6 +11,10 @@ PYLINT_DIRS = journalpump/ test/ systest/
 unittest:
 	$(PYTHON) -m pytest -vv test/
 
+.PHONY: pylint
+pylint:
+	$(PYTHON) -m pylint --rcfile .pylintrc $(PYLINT_DIRS) || exit $$(($$?&2))  # only fail on errors
```

Comment:
`pylint --enable F,E` ?

Fatal, error; warning is enabled by default too I think.

I would probably define this CI-only target to be error only - running the W and I checks and ignoring their results in CI doesn't make much sense (and default pylinting presumably is done elsewhere).

Reply:
because (tried) the --enable F,E has no effect on the return status